### PR TITLE
wp-cli 1.50

### PR DESCRIPTION
**Pull request by slushman**
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [X] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [X] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

-----

Updating WP-CLI to the 1.5.0 release.

### CODE REVIEW

File: Formula/wp-cli.rb [hunk 1]
```diff
@@ -3,26 +3,26 @@
 class WpCli < AbstractPhpPhar
   desc "Command-line tools for managing WordPress installations."
   homepage "https://wp-cli.org"
-  url "https://github.com/wp-cli/wp-cli/releases/download/v1.4.1/wp-cli-1.4.1.phar"
-  sha256 "325924cf161856f9478f2a154572698ecb5d1054e620843b9c43a7baf4e5ce3b"
+  url "https://github.com/wp-cli/wp-cli/releases/download/v1.5.0/wp-cli-1.5.0.phar"
+  sha512 "9385c63ab835c7c450529035cdb1f524b5878a67c7565c3497628e5ec4ec07ae4a34ef25c59a9e7d6edea7cdb039fcef7a1f731b922782b8c70418480bdff122"
```
Comment (SMillerDev) [inline Formula/wp-cli.rb]: This is not currently supported. Please stick with sha256.

File: Formula/wp-cli.rb [hunk 2]
```diff
@@ -3,26 +3,26 @@
...
   head "https://github.com/wp-cli/wp-cli.git"
 
   bottle do
-    cellar :any_skip_relocation
```
Comment (SMillerDev) [inline Formula/wp-cli.rb]: please don't edit the bottle block.

File: Formula/wp-cli.rb [hunk 3]
```diff
@@ -3,26 +3,26 @@
...
-    sha256 "70954b082e4b806ba0ca0f78e19bcad2d4ee641d28f13ab6a0e5df82db135b4a" => :el_capitan
+	cellar :any_skip_relocation
+	sha512 "9385c63ab835c7c450529035cdb1f524b5878a67c7565c3497628e5ec4ec07ae4a34ef25c59a9e7d6edea7cdb039fcef7a1f731b922782b8c70418480bdff122" => :high_sierra
+	sha512 "9385c63ab835c7c450529035cdb1f524b5878a67c7565c3497628e5ec4ec07ae4a34ef25c59a9e7d6edea7cdb039fcef7a1f731b922782b8c70418480bdff122" => :sierra
+	sha512 "9385c63ab835c7c450529035cdb1f524b5878a67c7565c3497628e5ec4ec07ae4a34ef25c59a9e7d6edea7cdb039fcef7a1f731b922782b8c70418480bdff122" => :el_capitan
   end
 
   def phar_file
-    "wp-cli-#{version}.phar"
+	"wp-cli-#{version}.phar"
```
Comment (SMillerDev) [inline Formula/wp-cli.rb]: can you leave the indentation as-is please?

**Comment by slushman** [comment]
I've updated the commit and renamed it. Sorry about the spacing, I think my editor did that.

**Comment by javian** [comment]
Thanks @slushman !

**Pull request closed without merging**